### DISCUSSION TITLE
Clipping planes

### DISCUSSION THREAD

**Discussion by jsbtic** [BradyAJohnston/MolecularNodes#606] [Q&A] [answered]
I wonder is it possible to use clipping planes  with molecular nodes to visualize cross sections etc? (cf. [VMD](https://www.ks.uiuc.edu/Research/vmd/plugins/cliptool/)) Normally I would try to use boolean modifier to apply a desired clip, but this would require converting the instances to mesh first(?)

My best

**Comment by BradyAJohnston** [comment, marked as answer]
I haven't used VMD so I'm not 100% sure of the result that you are after, but you can use the `Select Cube` to get a specific selection. Making the cube larger essentially gives you a selection on a plane:


https://github.com/user-attachments/assets/98734fff-db6c-447f-8b65-56a3fd9044b4

**Reply by jsbtic** [reply]
Much obliged, the `Select Cube` is much better than clipping plane as it allows arbitrary region selections. I did realize after posting the question that one can also achieve region selection based on distance, e.g. `resname SOL and prop y > 40`
![image](https://github.com/user-attachments/assets/553faf6c-715f-4e37-95e8-3b8b28472bfd)

**Reply by BradyAJohnston** [reply]
These string based selections are only available on MD trajectory objects (for now), but yes you can also achieve it with that. You could also combine with cube selection to have all sorts of control over it. Glad it's working for you!
